### PR TITLE
Build fixed

### PR DESCRIPTION
**Pull request by duzenko**
DXSDK not required
windows DPI setting is respected
Debug build fixed

### CODE REVIEW

**Comment by fholger** [comment]
What exactly was broken with the debug build? I believe I used it a couple times, and it ran fine.

File: sys/msvc/properties/_Debug.props [hunk 1]
```diff
@@ -16,7 +16,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>nafxcwd.lib;libcmtd.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <IgnoreSpecificDefaultLibraries>nafxcwd.lib;libcmtd.lib;%(IgnoreSpecificDefaultLibraries)</IgnoreSpecificDefaultLibraries>
+      <IgnoreSpecificDefaultLibraries>nafxcwd.lib;libcmt.lib;%(IgnoreSpecificDefaultLibraries)</IgnoreSpecificDefaultLibraries>
```
Comment (fholger) [inline sys/msvc/properties/_Debug.props:19]: Why?

File: sys/msvc/properties/_DoomDLL.props [hunk 2]
```diff
@@ -15,7 +15,7 @@
       <WarningLevel>Level3</WarningLevel>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>PolarSSL.lib;libcurl.lib;dbghelp.lib;dinput8.lib;dsound.lib;dxguid.lib;DxErr.lib;eaxguid.lib;glu32.lib;iphlpapi.lib;odbc32.lib;odbccp32.lib;opengl32.lib;winmm.lib;wsock32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>PolarSSL.lib;libcurl.lib;dbghelp.lib;dinput8.lib;dsound.lib;dxguid.lib;eaxguid.lib;glu32.lib;iphlpapi.lib;odbc32.lib;odbccp32.lib;opengl32.lib;winmm.lib;wsock32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```
Comment (fholger) [inline sys/msvc/properties/_DoomDLL.props:18]: So, is DxErr.lib really the only DX library that requires the SDK? That's kind of strange...

File: sys/msvc/properties/_Debug.props (hunk 1 above)
Comment (duzenko) [reply sys/msvc/properties/_Debug.props:19]: libcmt conflicts libcmtd for me in the debug build

File: sys/msvc/properties/_DoomDLL.props (hunk 2 above)
Comment (duzenko) [reply sys/msvc/properties/_DoomDLL.props:18]: others moved to winsdk in vs2013, this one was left behind

**Comment by duzenko** [comment]
It was giving me the "already defined in libcmt.lib" linker errors

File: sys/msvc/properties/_Debug.props (hunk 1 above)
Comment (fholger) [reply sys/msvc/properties/_Debug.props:19]: Ah, I see. There seems to be potential conflict if you pull in dependencies which are built against the release version...

**Pull request merged**